pkg/apis/kubeone/v1beta2/defaults.go

```diff
@@ -69,6 +70,15 @@ func SetDefaults_KubeOneCluster(obj *KubeOneCluster) {
 	SetDefaults_HelmReleases(obj)
 	SetDefaults_SystemPackages(obj)
 	SetDefaults_Features(obj)
+	SetDefaults_CloudConfig(obj)
+}
+
+func SetDefaults_CloudConfig(obj *KubeOneCluster) {
+	if obj.CloudProvider.AWS != nil {
```

Comment:
Shouldn't we do this only if external CCM is used? Or this should be used with the in-tree CCM as well?

Reply:
only external 
fixing...